src/more_ops.rs

```diff
@@ -462,6 +462,24 @@ pub fn op_substr<T: Allocator>(a: &mut T, input: T::Ptr, _max_cost: Cost) -> Res
     }
 }
 
+pub fn op_substr_r<T: Allocator>(a: &mut T, input: T::Ptr, _max_cost: Cost) -> Response<T::Ptr> {
+    let args = Node::new(a, input);
+    check_arg_count(&args, 2, "substr_r")?;
+    let a0 = args.first()?;
+    let s0 = atom(&a0, "substr")?;
+    let rest = args.rest()?;
+    let i1 = i32_atom(&rest.first()?, "substr")?;
```

Comment:
this was a typo. It would have been caught if we had that unit test in `clvm` adam :)